code/modules/reagents/reagent_containers/glass_containers.dm

```diff
@@ -246,7 +246,7 @@
 	else
 		usr << "<span class='notice'>There is no assembly to remove.</span>"
 
-/obj/item/weapon/reagent_containers/glass/beaker/ignite()
+/obj/item/weapon/reagent_containers/glass/beaker/fire_act()
```

Comment:
this is supposed to be ignite--as it's a special proc only for beakers--if you need to rename it, that's fine, but it's definitely not supposed to be fire_act